### PR TITLE
Feature/renew k8s version

### PR DESCRIPTION
**Pull request by takama**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.retry
 community
 gce
+openprovider
```
Comment (rumyantseva) [inline .gitignore:4]: Зачем упоминаем openprovider здесь?

Comment (takama) [reply .gitignore:4]: How to ignore folder with inventory file? Do you mean we need another name? What is bad with this name of the folder?

File: playbooks/system/update.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      name: '*'
+      state: latest
+    notify:
+      - reboot vps
+      - wait to come up
+ 
+  handlers:
+
+    - name: reboot vps
+      command: /sbin/shutdown -r +1
```
Comment (STASiAN) [inline playbooks/system/update.yml:21]: why do you use +1 ???
shutdown -r now

from manual:
"now" is an alias for "+0", i.e. for triggering an immediate shutdown.
If no time argument is specified, "+1" is implied.

File: .gitignore (hunk 1 above)
Comment (rumyantseva) [reply .gitignore:4]: Well, ok. Let's use it If I have a better idea, I'll tell you

File: playbooks/system/update.yml (hunk 2 above)
Comment (takama) [reply playbooks/system/update.yml:21]: It needs for ansible playbook, which catch ssh disconnect before async attribute work

**Review by rumyantseva — APPROVED** [review]

**Review by STASiAN — APPROVED** [review]

**Pull request merged**